mocks/handlers/auth.ts

```diff
@@ -0,0 +1,92 @@
+import { CognitoUserAttributes } from "shared-types";
+import { isCmsUser } from "shared-utils";
+import {
+  CognitoUserResponse,
+  makoReviewer,
+  makoStateSubmitter,
+  userResponses,
+} from "../data/users";
+
+export const setMockUsername = (user?: string | CognitoUserResponse | null): void => {
+  if (user) {
+    if (typeof user === "string") {
+      process.env.MOCK_USER_USERNAME = user;
+    } else {
+      process.env.MOCK_USER_USERNAME = user.Username;
+    }
+  } else {
+    delete process.env.MOCK_USER_USERNAME;
+  }
+};
+
+export const useDefaultStateSubmitter = () => setMockUsername(makoStateSubmitter);
+
+export const useDefaultReviewer = () => setMockUsername(makoReviewer);
```

Comment:
I like the helper methods. It makes it more declarative as to what you are doing